### PR TITLE
Vet clinic database: query multiple tables

### PR DESCRIPTION
**Pull request by Fombi-Favour**
In this activity, we:
- created `owners` and `species` tables
- updated the `animals` table
- made use of the foreign keys

## Queries
- >What animals belong to Melody Pond?
project3-sc1
- >List of all animals that are pokemon (their type is Pokemon).
project3-sc2
- >List all owners and their animals, remember to include those that don't own any animal.
project3-sc3
- >How many animals are there per species?
project3-sc4
- >List all Digimon owned by Jennifer Orwell.
project3-sc5
- >List all animals owned by Dean Winchester that haven't tried to escape.
project3-sc6
- >Who owns the most animals?
project3-sc7

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -10,4 +10,26 @@ INSERT INTO animals(name, date_of_birth, escape_attempts, neutered, weight_kg) V
...
+INSERT INTO owners (full_name, age) VALUES('Dean Winchester', 14);
+INSERT INTO owners (full_name, age) VALUES('Jodie Whittaker', 38);
+
+-- inserting data in "species" table
+INSERT INTO species (name) VALUES('Pokemon');
+INSERT INTO species (name) VALUES('Digimon');
+
+-- updating data in "animals" table
+UPDATE animals SET species_id=2 WHERE name LIKE '%mon';
+UPDATE animals SET species_id=1 WHERE species_id IS NULL;
```
Comment (skyv26) [inline data.sql]: - [ ] Dear @Fombi-Favour you did a great job, and your query is working fine. But I think your approach is not correct, you are trying to update things by adding the id statically, just think about a scenario, where you need to make the changes but you don't have any idea about the `ID` of the species. Kindly look at the below hint to understand to update the column data more efficiently.

## Hint
image

**Review by skyv26 — CHANGES_REQUESTED** [review]
## Hi @Fombi-Favour, 

Good job so far 👍🏻 
There are some issues that you still need to work on to go to the next project but you are almost there!

## STATUS: REQUIRE CHANGES 🔴 



## Highlights
- Good **PR** title ✅ 
- Descriptive *PR* description ✅ 
- Implement the features ✅ 


## Required Changes ♻️

- [ ] Kindly remove the unnecessary `things` from your `README` file, you already have added the `getting-started` section at the appropriate place. So that's why you don't need to add it again at the top of the `README` file table of contents. I would like to suggest you to either add the logo relevant to the project theme or delete it entirely.

image

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._


#### Cheers, and Happy coding!👏👏👏

## ⚠️  WARNING ⚠️ 
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

## Have any doubt ❓ 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @skyv26 in your question so I can receive the notification. You can also follow me on the below platforms

<table>
<tr>
<td>

<a href="https://github.com/skyv26">
   @skyv26
</a>

</td>
<td>

<a href="https://www.linkedin.com/in/skyv2022/">
   linkedin-skyv2022
</a>

</td>
<td>

<a href="https://twitter.com/vrma_aakash">
   @vrma_aakash
</a>

</td>
<td>

<a href="https://www.instagram.com/skybrel/">
   @skybrel
</a>

</td>
<td>

<a href="https://www.frontendmentor.io/profile/skyv26">
   @skybrel
</a>

</td>
</tr>
</table>

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by AbelG101 — APPROVED** [review]
# Hi @Fombi-Favour 👋, 

**Your project is complete!  There is nothing else to say other than... it's time to merge it. 
Congratulations! 🎉 🥳🥇**

## Highlights
- Correct queries for each question ✔
- Proper use of GitFlow ✔ 
- Descriptive PR title with a great organized summary ✔

## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by zeff96 — COMMENTED** [review]
### Status: APPROVED :green_circle: 
Hi @Fombi-Favour , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

congrats

## Highlights
:heavy_check_mark: Descriptive PR
:heavy_check_mark: Correct GitFlow


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**